infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/model/KubernetesRuntimeState.java

```diff
@@ -13,26 +13,31 @@
 
 import java.util.Objects;
 import javax.persistence.Column;
-import javax.persistence.Embeddable;
-import javax.persistence.EmbeddedId;
 import javax.persistence.Entity;
+import javax.persistence.Id;
 import javax.persistence.NamedQueries;
 import javax.persistence.NamedQuery;
 import javax.persistence.Table;
 import org.eclipse.che.api.core.model.workspace.WorkspaceStatus;
 import org.eclipse.che.api.core.model.workspace.runtime.RuntimeIdentity;
+import org.eclipse.che.api.workspace.server.model.impl.RuntimeIdentityImpl;
 
 /** @author Sergii Leshchenko */
 @Entity(name = "KubernetesRuntime")
 @Table(name = "che_k8s_runtime")
 @NamedQueries({
-  @NamedQuery(name = "KubernetesRuntime.getAll", query = "SELECT r FROM KubernetesRuntime r"),
-  @NamedQuery(
-      name = "KubernetesRuntime.getByWorkspaceId",
```

Comment:
@sleshchenko wdyt?

Reply:
@skabashnyuk What is a question? It's OK for me to remove this named query since workspace id became the primary key.

Reply:
yes

Reply:
@sleshchenko for now workspaceId+ownerId is the primary key. But I believe it doesn't change what you wanted to say, right?

Reply:
@garagatyi Thanks for the clarification. Right, it doesn't change. I meant JPA ID (javax.persistence.Id) but not Database primary key.